docs/client-concepts/high-level/indexing/indexing.asciidoc

```diff
@@ -0,0 +1,172 @@
+:ref_current: https://www.elastic.co/guide/en/elasticsearch/reference/6.x
+
+:github: https://github.com/elastic/elasticsearch-net
+
+:nuget: https://www.nuget.org/packages
+
+////
+IMPORTANT NOTE
+==============
+This file has been generated from https://github.com/elastic/elasticsearch-net/tree/6.x/src/Tests/Tests/ClientConcepts/HighLevel/Indexing/Indexing.doc.cs. 
+If you wish to submit a PR for any spelling mistakes, typos or grammatical errors for this file,
+please modify the original csharp file found at the link and submit the PR with that change. Thanks!
+////
+
+[[indexing]]
+=== Indexing
+
+NEST has a number of ways in which documents can be indexed.
+
+==== Single Documents
+
+A single document can be indexed at a time, either synchronously or asynchronously
+
+[source,csharp]
+----
+var person = new Person
+{
+    Id = 1,
+    FirstName = "Martijn",
+    LastName = "Laarman"
+};
+
+var indexResponse = client.IndexDocument(person); <1>
+
+var asyncIndexResponse = await client.IndexDocumentAsync(person); <2>
+----
+<1> synchronous method that returns an `IIndexResponse`
+
+<2> asynchronous method that returns a `Task<IIndexResponse>` that can be awaited
+
+==== Multiple Documents
+
+Multiple documents can be indexed using the bulk methods, again either synchronously or asynchronously
+
+[source,csharp]
+----
+var people = new []
+{
+    new Person
+    {
+        Id = 1,
+        FirstName = "Martijn",
+        LastName = "Laarman"
+    },
+    new Person
+    {
+        Id = 2,
+        FirstName = "Stuart",
+        LastName = "Cam"
+    },
+    new Person
+    {
+        Id = 3,
+        FirstName = "Russell",
+        LastName = "Cam"
+    }
+};
+
+var bulkIndexResponse = client.IndexMany(people); <1>
```

Comment:
split out `IndexMany<T>()` into a separate section; it's client specific shortcut to `_bulk` so I think this should be called out. It is more limited than using `Bulk()`  directly, but handles the simple case.

There should also be a similar section for `.Bulk(b => b.IndexMany()`, then leading into a section on `Bulk`, I think.

Reply:
Great suggestions.